generation/scraper/Partitions/DirectXShaderCompiler/main.cpp

```diff
@@ -0,0 +1,10 @@
+#define SECURITY_WIN32 // For sspi.h
+#define QCC_OS_GROUP_WINDOWS
+
+#include "intrinfix.h"
+
+#include "windows.fixed.h"
+#include <sdkddkver.h>
+
+#include <dxcapi.h>
+// #include <dxcisense.h>
```

Comment:
Unfortunately the IntelliSense headers aren't bundled (yet?) but those woudl be needed to fully convert [hassle-rs](https://github.com/Traverse-Research/hassle-rs/) over to `windows-rs`.

Perhaps those can be included separately when it's easier to generate [loose metadata files](https://github.com/microsoft/windows-rs/tree/master/.windows/winmd) following #414.